src/index.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable */
```

Comment:
I think it is better to fix linters issues rather than disabling the linters.
https://github.com/muneebulrehman/todo-review/blob/582e368b6867ed8f4e7683e7bd3ffe52f964ef53/src/index.js#L1
https://github.com/muneebulrehman/todo-review/blob/582e368b6867ed8f4e7683e7bd3ffe52f964ef53/src/index.js#L29